scenes/quests/lore_quests/quest_003/ink_drinker_levels/ink_combat_round_4.tscn

```diff
@@ -0,0 +1,207 @@
+tile_set = ExtResource("1_7jrc5")
+
+[node name="Stones" type="TileMapLayer" parent="TileMapLayers"]
+tile_map_data = PackedByteArray("AAAAAAAABAABAAMAAAABAAAABAABAAMAAAACAAAABAABAAMAAAADAAAABAACAAMAAAAOAP//BAACAAEAAAD//wAABAAAAAEAAAD//wEABAAAAAEAAAD//wIABAAAAAEAAAD//wMABAAAAAEAAAASAP//BAABAAAAAAARAP//BAABAAAAAAAQAP//BAABAAEAAAAPAP//BAAAAAEAAAASAAAABAACAAEAAAARAAAABAABAAEAAAAQAAAABAABAAMAAAACAAsABAAAAAMAAAADAAsABAABAAMAAAAEAAsABAABAAMAAAAFAAsABAABAAMAAAAGAAsABAABAAMAAAAKAAsABAABAAMAAAALAAsABAABAAMAAAAMAAsABAABAAMAAAANAAsABAABAAMAAAAOAAsABAABAAMAAAAPAAsABAACAAMAAAAPAAoABAACAAAAAAAOAAoABAABAAQAAAANAAoABAABAAAAAAAMAAoABAABAAQAAAALAAoABAABAAAAAAAKAAoABAABAAQAAAAJAAoABAABAAAAAAAHAAoABAABAAAAAAAGAAoABAABAAQAAAAFAAoABAABAAAAAAAEAAoABAABAAQAAAADAAoABAABAAAAAAACAAoABAAAAAQAAAAPAAAABAABAAMAAAD//wQABAAAAAEAAAASAAEABAACAAEAAAARAAEABAAAAAEAAAARAAIABAAAAAEAAAASAAIABAACAAEAAAAOAAAABwABAAMAAAARAAMABAAAAAEAAAASAAMABAABAAAAAAARAAQABAAAAAEAAAASAAQABAADAAIAAAAFAAAABAAAAAMAAAARAAgABAADAAAAAAAGAAAABwABAAAAAAAHAAAABAACAAMAAAAJAAAABAAAAAMAAAAKAAAABwABAAAAAAALAAAABAACAAMAAAANAAAABAAAAAMAAAAIAAoABAABAAAAAAD/////BAAAAAAAAAAAAP//BAABAAAAAAABAP//BAABAAAAAAACAP//BAABAAQAAAADAP//BAABAAQAAAAFAP//BAABAAAAAAAGAP//BAABAAQAAAAHAP//BAABAAIAAAAJAP//BAABAAQAAAAKAP//BAABAAIAAAALAP//BAACAAIAAAANAP//BAAAAAIAAAD//wUABwACAAIAAAD//wYABAACAAMAAAD//wgABAADAAAAAAD//wkABAAAAAIAAAAAAAkABAACAAAAAAD//woABAAAAAMAAAAAAAoABAACAAMAAAAHAAsABAABAAMAAAAIAAsABAABAAMAAAAJAAsABAABAAMAAAASAAoABAACAAMAAAARAAoABwAAAAUAAAARAAkABAAAAAIAAAASAAkABAACAAAAAAARAAYABAADAAMAAAARAAUABAAAAAIAAAASAAUABwAAAAMAAAAKAAEABAADAAMAAAAGAAEABAADAAMAAAAOAAIABAADAAAAAAAOAAMABAADAAIAAAAOAAQABwADAAUAAAAOAAgABAADAAMAAAACAAcABAADAAIAAAACAAgABwADAAUAAAACAAYABAADAAAAAAAJAAUABAADAAAAAAAJAAYABAACAAIAAAAIAAgABAADAAMAAAAJAAcABAABAAMAAAAKAAcABAACAAMAAAAKAAYABAACAAAAAAAHAAYABAAAAAQAAAAHAAcABAAAAAMAAAAIAAYABAABAAAAAAAIAAcABAABAAIAAAATAAQABAABAAEAAAATAAMABAABAAEAAAATAAIABAABAAEAAAATAAEABAABAAEAAAATAAAABAABAAEAAAATAP//BAACAAAAAAATAAUABAACAAMAAAACAAIABAADAAQAAAACAAMABAADAAUAAAAOAAcABAADAAQAAAA=")
+tile_set = ExtResource("1_7jrc5")
+
+[node name="TileMapLayer" type="TileMapLayer" parent="TileMapLayers"]
+tile_set = ExtResource("1_7jrc5")
+
+[node name="OnTheGround" type="Node2D" parent="."]
+y_sort_enabled = true
+
+[node name="LevelExit" type="Node2D" parent="OnTheGround"]
+y_sort_enabled = true
+position = Vector2(546, -2)
+
+[node name="Door Exit" parent="OnTheGround/LevelExit" instance=ExtResource("2_jwan6")]
+position = Vector2(0, 82)
+play_victory_fanfare_on_open = true
+
+[node name="Teleporter" type="Area2D" parent="OnTheGround/LevelExit"]
+position = Vector2(0, -25)
+collision_layer = 4
+script = ExtResource("6_h1kdf")
+next_scene = "uid://bo6qfusag3fae"
```

Comment:
When I finish the level, I am teleported to the 3rd round of the first lore quest. To keep the quest playable, it should go to Fray's End (`scenes/quests/lore_quests/quest_003/ink_drinker_levels/ink_combat_round_4.tscn`).